.travis.yml

```diff
@@ -41,8 +47,11 @@ matrix:
         - MATRIX_EVAL="CC=gcc-6 && CXX=g++-6"
 
     - compiler: clang
+      os: linux
       env: TASKS="ctest clang-3.9"
 
+    - os: osx
+
```

Comment:
Adding a TASKS="macOS C++" or similar would make the matrix easier to read.

Reply:
Sure.